index.js

```diff
@@ -4,6 +4,9 @@ var hasFlag = require('has-flag');
 var serializeError = require('serialize-error');
 var Runner = require('./lib/runner');
 var runner = new Runner();
+var cache = require('cacha')('.ava/cache');
+
+cache.clean();
```

Comment:
`cache.clean()` removes outdated files from cache directory (so it will not grow endlessly). Function name a little misleading, true.